docs/DXIL.rst

```diff
@@ -3038,6 +3038,7 @@ DECL.EXTRAARGS                            Extra arguments not allowed for shader
 DECL.FNATTRIBUTE                          Functions should only contain known function attributes
 DECL.FNFLATTENPARAM                       Function parameters must not use struct types
 DECL.FNISCALLED                           Functions can only be used by call instructions
+DECL.MESHNODEOUTPUTRECORD                 A mesh shader may not have any output records,
```

Comment:
I wrote a comment that I didn't like the wording here, but it seems someone beat me to it and this is stale, which is causing the CI build failures.